### PR TITLE
[Version] Bump version to v0.6.0

### PR DESCRIPTION
**Pull request by HaiyiMei**
- Fix the reloading problem of `RPCFactory` bf9db43a4044f6cf3a81284e38650264171516f6 75cc595f45b85f0370302ace9c356e1fe1022e1b ccd3fd5df0e25730ca2557102f515833ca063b40
- Fix camera parameter exported by `CameraParamter` 312cc34d2972bd806aa9a776a6f411d351efb554
- Optimize `MeshOpertator` in UE 254d7ec81e3e533fb244d37dfa8c51f092c64c58
- Move anim from `samples.anim` to `xrfeitoria.utils.anim` 8c0b7f920b40646fbc8f7e7f3bcae6929e60327c
- Update CLI 9ebc0e375dff15c467d20631ac4da430696a0653
	- move cmd to cmd/blender
	- add `xf-install-plugin`
- Add `Material` 923a197e14f9ca26fe836c4969688126e163f69d
- Add `reload_rpc_code` in `xf.init_blender(reload_rpc_code=...)` or `xf.init_unreal(reload_rpc_code=...)` 4d004780cfd39b1a4920db8ca941ebb4022c7dbe
- Polish `setup_logger` 5336af47567ac7ea397a573b36035cee61c00365
- Add more levels in logger, 'RPC' < 'TRACE' < 'DEBUG' < 'INFO' 4d004780cfd39b1a4920db8ca941ebb4022c7dbe
	- 'RPC' (custom level): logging RPC messages which are sent by RPC protocols.
	- 'TRACE': logging engine output like console output of blender.
	- 'DEBUG': logging debug messages.
	- 'INFO': logging info messages.
- Add `dump_humandata` for `SMPLMotion` and `SMPLXMotion` 6de395e5dd8807ef8615ca86c5fd0224f2ea31b5
- Optimize version control 5defe13ee76a06ec598a7c0fd0c239cd61e3bf74 d762cec7cc0b09f7083c40d4a596f678b9aa1f74 c7371756bce7b2b9b8338ea16ed17b28aca67750
- Add instructions on building/publishing your own plugins c7371756bce7b2b9b8338ea16ed17b28aca67750 eb1c5c74dc91f3287ba93136d9753da04a089a28
- Add support for exporting audio for Unreal 919aa41f6891c2db79f26f89344c8dd17dd0f2d8
- Optimize Docs

### CODE REVIEW

File: xrfeitoria/sequence/sequence_unreal.py [hunk 1]
```diff
@@ -148,6 +169,7 @@ def spawn_actor_with_keys(
         actor_name: Optional[str] = None,
         stencil_value: int = 1,
         anim_asset_path: 'Optional[str]' = None,
+        motion_data: 'Optional[]' = None,
```
Comment (Maoxie) [inline xrfeitoria/sequence/sequence_unreal.py]:
```suggestion
        motion_data: 'Optional[List[MotionFrame]]' = None,
```

File: samples/unreal/07_amass.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+# 2.1 Download SMPL-XL model from https://openxrlab-share.oss-cn-hongkong.aliyuncs.com/xrfeitoria/assets/SMPL-XL-001.fbx
+# or from https://openxlab.org.cn/datasets/OpenXDLab/SynBody/tree/main/Assets
+# With downloading this, you are agreeing to CC BY-NC-SA 4.0 License (https://creativecommons.org/licenses/by-nc-sa/4.0/).
+smpl_xl_file = root / 'SMPL-XL-001.fbx'
+# 2.2 Download the meta information from https://openxrlab-share.oss-cn-hongkong.aliyuncs.com/xrfeitoria/assets/SMPL-XL-001.npz
+smpl_xl_meta_file = root / 'SMPL-XL-001.npz'
+
+# 3. Define the output file path
+seq_name = 'seq_amass'
+output_path = Path(__file__).parents[2].resolve() / 'output/samples/unreal' / Path(__file__).stem
```
Comment (Maoxie) [inline samples/unreal/07_amass.py:38]:
```suggestion
output_path = Path(__file__).resolve().parents[2] / 'output/samples/unreal' / Path(__file__).stem
```

File: xrfeitoria/cmd/blender/vis_smplx.py [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+    from xrfeitoria.utils.anim.motion import Motion, SMPLMotion, SMPLXMotion
+except (ImportError, NameError):
+    pass
+
+app = Typer(pretty_exceptions_show_locals=False)
+
+
+@remote_blender()
+def add_smplx(betas: 'Tuple[float, ...]' = [0.0] * 10, gender: str = 'neutral') -> str:
```
Comment (Maoxie) [inline xrfeitoria/cmd/blender/vis_smplx.py:28]: Use mutable types (list) as default values could cause bugs.
```suggestion
def add_smplx(betas: 'Tuple[float, ...]' = tuple([0.0] * 10), gender: str = 'neutral') -> str:
```

File: xrfeitoria/renderer/renderer_blender.py [hunk 4]
```diff
@@ -24,14 +24,15 @@
 from .renderer_base import RendererBase, render_status
 
 try:
-    from ..data_structure.models import RenderJobBlender, RenderPass  # isort:skip
+    # only for linting, not imported in runtime
```
[the remark this answers is not in the document]
Comment (HaiyiMei) [reply xrfeitoria/renderer/renderer_blender.py:27]: fixed in f93d86b1b32bd33c3591db273416b4e56a000f7d

File: xrfeitoria/sequence/sequence_unreal.py (hunk 1 above)
Comment (HaiyiMei) [reply xrfeitoria/sequence/sequence_unreal.py]: fixed in f93d86b1b32bd33c3591db273416b4e56a000f7d

File: src/XRFeitoriaBpy/core/factory.py [hunk 5]
```diff
@@ -133,29 +135,39 @@ def set_sequence_properties(
...
         collection.sequence_properties.level = level
         collection.sequence_properties.fps = fps
         collection.sequence_properties.frame_start = frame_start
         collection.sequence_properties.frame_end = frame_end
         collection.sequence_properties.frame_current = frame_current
+        collection.sequence_properties.resolution_x = resolution_x
+        collection.sequence_properties.resolution_y = resolution_y
 
-    def get_sequence_properties(collection: 'bpy.types.Collection') -> 'Tuple[bpy.types.Scene, int, int, int, int]':
+    def get_sequence_properties(
```
[the remark this answers is not in the document]
Comment (HaiyiMei) [reply src/XRFeitoriaBpy/core/factory.py]: Good point, resolved in [`58f1ada` (#15)](https://github.com/openxrlab/xrfeitoria/pull/15/commits/58f1ada79c17df94c10a55633a41dc965a4a2816)

File: src/XRFeitoriaUnreal/Content/Python/utils.py [hunk 6]
```diff
@@ -88,66 +91,80 @@ def import_asset(path: Union[str, List[str]], dst_dir_in_engine: Optional[str] =
...
+            import_task.set_editor_property('destination_name', '')
+            import_task.set_editor_property('destination_path', dst_dir)
+            import_task.set_editor_property('filename', path)
+            import_task.set_editor_property('replace_existing', replace)
+            import_task.set_editor_property('options', import_options)
+
+            import_tasks = [import_task]
+            asset_tools.import_asset_tasks(import_tasks)
+            asset_paths.extend(
+                [path.split('.')[0] for path in import_task.get_editor_property('imported_object_paths')]
```
[the remark this answers is not in the document]
Comment (HaiyiMei) [reply src/XRFeitoriaUnreal/Content/Python/utils.py]: Fixed in [`58f1ada` (#15)](https://github.com/openxrlab/xrfeitoria/pull/15/commits/58f1ada79c17df94c10a55633a41dc965a4a2816)

File: src/XRFeitoriaUnreal/Content/Python/utils.py [hunk 7]
```diff
@@ -88,66 +91,80 @@ def import_asset(path: Union[str, List[str]], dst_dir_in_engine: Optional[str] =
...
+            )
+        else:
+            assetsTools = unreal.AssetToolsHelpers.get_asset_tools()
+            assetImportData = unreal.AutomatedAssetImportData()
+            assetImportData.destination_path = dst_dir
+            assetImportData.filenames = [path]
+            assetImportData.replace_existing = replace
+            assets: List[unreal.Object] = assetsTools.import_assets_automated(assetImportData)
+            asset_paths.extend([asset.get_path_name().split('.')[0] for asset in assets])
```
[the remark this answers is not in the document]
Comment (HaiyiMei) [reply src/XRFeitoriaUnreal/Content/Python/utils.py]: [`58f1ada` (#15)](https://github.com/openxrlab/xrfeitoria/pull/15/commits/58f1ada79c17df94c10a55633a41dc965a4a2816#diff-69a9f185d2d4203375733707c84b1c935c5d326dd6698ec8cbf1de2a875debb7R119)

File: xrfeitoria/renderer/renderer_blender.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (HaiyiMei) [reply xrfeitoria/renderer/renderer_blender.py]: Sorry for the misunderstanding, this part is being used by code sent to blender. 
I changed the comment to `# linting and for engine` in [`58f1ada` (#15)](https://github.com/openxrlab/xrfeitoria/pull/15/commits/58f1ada79c17df94c10a55633a41dc965a4a2816#diff-71c6850dca8dffb4006e3078800162bfa32e3ddfb8d10dda05e1f18dc6816313R11)

File: samples/unreal/07_amass.py (hunk 2 above)
Comment (HaiyiMei) [reply samples/unreal/07_amass.py]: Accepted in [`58f1ada` (#15)](https://github.com/openxrlab/xrfeitoria/pull/15/commits/58f1ada79c17df94c10a55633a41dc965a4a2816#diff-050f51ccfd09bce14b1e89a956edfd76f9a5097030783dc6c6970db03c6e208fR38)

File: xrfeitoria/cmd/blender/vis_smplx.py (hunk 3 above)
Comment (HaiyiMei) [reply xrfeitoria/cmd/blender/vis_smplx.py]: Fixed in [`58f1ada` (#15)](https://github.com/openxrlab/xrfeitoria/pull/15/commits/58f1ada79c17df94c10a55633a41dc965a4a2816#diff-c4f7228b2fbf2d2bf2ef2406413575841160c9320874219b14282abe778e605dR28).
Modify it to `betas: 'Tuple[float, ...]' = (0.0,) * 10`

File: src/XRFeitoriaUnreal/Content/Python/utils.py [hunk 8]
```diff
@@ -88,66 +91,80 @@ def import_asset(path: Union[str, List[str]], dst_dir_in_engine: Optional[str] =
...
-        import_task.set_editor_property('destination_path', dst_dir)
-        import_task.set_editor_property('filename', path)
-        import_task.set_editor_property('replace_existing', True)
-        import_task.set_editor_property('options', import_options)
-
-        import_tasks = [import_task]
-        asset_tools.import_asset_tasks(import_tasks)
-        asset_paths.extend([path.split('.')[0] for path in import_task.get_editor_property('imported_object_paths')])
-
+        if path.endswith('.fbx'):
```
[the remark this answers is not in the document]
Comment (HaiyiMei) [reply src/XRFeitoriaUnreal/Content/Python/utils.py]: Fixed in [`182d4e9` (#15)](https://github.com/openxrlab/xrfeitoria/pull/15/commits/182d4e9cc9366ffdde10e1105c1fb7a622c14ee7)

File: samples/unreal/07_amass.py (hunk 2 above)
Comment (HaiyiMei) [reply samples/unreal/07_amass.py]: Modified all the similar situations [`17a3e7a` (#15)](https://github.com/openxrlab/xrfeitoria/pull/15/commits/17a3e7aec1fb07bb619b36e2739adad91903b6d9)

File: src/XRFeitoriaBpy/core/factory.py [hunk 9]
```diff
@@ -133,29 +145,35 @@ def set_sequence_properties(
...
         level = collection.sequence_properties.level
         fps = collection.sequence_properties.fps
         frame_start = collection.sequence_properties.frame_start
         frame_end = collection.sequence_properties.frame_end
         frame_current = collection.sequence_properties.frame_current
-        return level, fps, frame_start, frame_end, frame_current
+        resolution_x = collection.sequence_properties.resolution_x
+        resolution_y = collection.sequence_properties.resolution_y
+
+        return level, fps, frame_start, frame_end, frame_current, resolution_x, resolution_y
```
Comment (Maoxie) [inline src/XRFeitoriaBpy/core/factory.py:176]: A class inheriting NamedTuple is not only a typing, but can also be used to initialize an instance.

```suggestion
        return SequenceProperties(level, fps, frame_start, frame_end, frame_current, resolution_x, resolution_y)
```

File: xrfeitoria/utils/publish_plugins.py [hunk 10]
```diff
@@ -1,98 +1,191 @@
...
 
 
 def _make_archive(
-    plugin_folder: PathLike,
-    zip_name: Optional[str] = None,
-    folder_name: Optional[str] = None,
+    src_folder: Path,
+    dst_path: Optional[Path] = None,
+    folder_name_inside_zip: Optional[str] = None,
+    filter_names: List[str] = ['.git', '.idea', '.vscode', '.gitignore', '.DS_Store', '__pycache__', 'Intermediate'],
```
Comment (Maoxie) [inline xrfeitoria/utils/publish_plugins.py:42]: Using a mutable default value as an argument is not recommended.
Considering one of the following approaches:
* Defaults to None, and `if filter_names is None: filter_names = [...]`
* Defaults to Tuple

File: src/XRFeitoriaBpy/core/factory.py (hunk 9 above)
Comment (HaiyiMei) [reply src/XRFeitoriaBpy/core/factory.py]: Accepted in [`6ec2877` (#15)](https://github.com/openxrlab/xrfeitoria/pull/15/commits/6ec287755d8d27ec5a11e8578c65e2581ffc02c9)

File: xrfeitoria/utils/publish_plugins.py (hunk 10 above)
Comment (HaiyiMei) [reply xrfeitoria/utils/publish_plugins.py]: Accepted in [`9dca9a9` (#15)](https://github.com/openxrlab/xrfeitoria/pull/15/commits/9dca9a9d5c62d5cb66b3c1a5b226f1ee5282aa65)

**Pull request merged**